src/QirRuntime/lib/QIR/bridge-qis.ll

```diff
@@ -336,3 +347,10 @@ define double @__quantum__qis__arctan2__body(double %y, double %x) {  ; Q#: func
   %result = call double @quantum__qis__arctan2__body(double %y, double %x)
   ret double %result
 }
+
+; operation DrawRandomInt (min : Int, max : Int) : Int      
+; https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.random.drawrandomint

```

Comment:
[nit] In general, en-us should be dropped from documentation URLs as that has the side effect of overwriting users' language preferences.

```suggestion
; https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.random.drawrandomint

```

Reply:
Wow, what a curious functionality!.. Thank you for sharing the knowledge.
Fixed this in the subsequent added changes I'm working on.